app/assets/javascripts/discourse/app/components/emoji-picker.js

```diff
@@ -245,15 +255,152 @@ export default Component.extend({
 
   @action
   keydown(event) {
+    const arrowKeys = ["ArrowDown", "ArrowUp", "ArrowLeft", "ArrowRight"];
```

Comment:
We need to add some tests for the arrow behaviour. 

You can use the `triggerKeyEvent` test helper to simulate keyboard presses in tests. We have some similar tests in composer or search acceptance tests.